addons/mail/static/src/scss/variables/primary_variables.scss

```diff
@@ -29,3 +29,4 @@ $o-mail-discuss-sidebar-category-item-avatar-size: 30px !default;
 $o-mail-discuss-sidebar-scrollbar-width: 15px !default;
 
 $o-mail-message-sidebar-width: 50px;
+$o-mail-composer-text-input-height: 32px !default;
```

Comment:
Before vs after
<img width="2250" alt="Screenshot 2022-05-16 at 15 11 42" src="https://user-images.githubusercontent.com/6569390/168601403-d059e3b0-6fc5-4c17-bb03-94d8b7375d92.png">
<img width="970" alt="Screenshot 2022-05-16 at 15 12 29" src="https://user-images.githubusercontent.com/6569390/168601412-c7fbed0e-9c19-471b-884f-2be29aa98660.png">

I get the issue that is being resolved, but I think this is not the correct way.
Shrinking the height of composer makes the composer too small in height.

Reply:
Hey @alexkuhn 👋 
I can understand your feeling about the input height, so I tried to find another way to fix this alignment inconsistency without changing the height.
My new approach is to center the avatar vertically with the input. This solution is much simpler in terms of code and solves this vertical inconsistency, what do you think?

cc. @stefanorigano

Reply:
> My new approach is to center the avatar vertically with the input. This solution is much simpler in terms of code and solves this vertical inconsistency, what do you think?

You mean the avatar is always centred, including composer height of 3 text lines and the avatar is aligned with 2nd line?

The right buttons stretch, which is not pretty but the best tradeoff with the overall layout with pretty border around input and the buttons.

IMO the elements of composer should stick to the top of the component.
For reference, Discord does that with all buttons in their composer, and it feels natural that way.